website/static/js/pages/file-page.js

```diff
@@ -4,4 +4,44 @@ var $osf = require('js/osfHelpers');
 var FileViewPage = require('js/filepage');
 var waterbutler = require('js/waterbutler');
 
+require('jquery-tagsinput');
+
 m.mount(document.getElementsByClassName('file-view-panels')[0], FileViewPage(window.contextVars));
+
+function build_tag_url(nid, filePath, tag) {
+    return '/api/v1/project/' + nid + '/osfstorage' + filePath + '/tags/' + tag + '/';
+}
+
+$(function() {
```

Comment:
This is almost the same code as https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/pages/project-dashboard-page.js#L118-#L148. Would it be possible to refactor this into a more general function that can be used in both places?